src/meta/src/rpc/server.rs

```diff
@@ -352,12 +378,31 @@ pub async fn start_service_as_election_leader(
     mut svc_shutdown_rx: WatchReceiver<()>,
 ) -> MetaResult<()> {
     tracing::info!("Defining leader services");
+    if let Some(sql_store) = &meta_store_sql {
```

Comment:
Check and do auto upgrade when meta starts. It will run any migration plans if found in this version. It will ignore if all plans are applied already.